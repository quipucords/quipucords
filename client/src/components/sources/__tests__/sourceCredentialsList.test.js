import React from 'react';
import { mount } from 'enzyme';
import SourceCredentialsList from '../sourceCredentialsList';

describe('SourceCredentialsList Component', () => {
  it('should render a sorted list', () => {
    const props = {
      source: {
        credentials: [
          {
            name: 'a test'
          },
          {
            name: 'b test'
          }
        ]
      }
    };

    const component = mount(<SourceCredentialsList {...props} />);

    expect(component.render()).toMatchSnapshot();
  });
});
