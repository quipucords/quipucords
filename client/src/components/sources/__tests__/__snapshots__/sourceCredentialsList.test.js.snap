// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SourceCredentialsList Component should render a sorted list 1`] = `
<div
  class="container-fluid"
>
  <div
    class="row"
  >
    <div
      class="col-sm-4 col-xs-12"
    >
      <span>
        <span
          aria-hidden="true"
          class="fa fa-id-card"
        />
          a test
      </span>
    </div>
  </div>
  <div
    class="row"
  >
    <div
      class="col-sm-4 col-xs-12"
    >
      <span>
        <span
          aria-hidden="true"
          class="fa fa-id-card"
        />
          b test
      </span>
    </div>
  </div>
</div>
`;
